run_demo.sh

```diff
@@ -134,11 +134,12 @@ fi
 # address/IP-address.
 machine_hostname=$(hostname | tr '[:upper:]' '[:lower:]')
 if ! check_hostname "${machine_hostname}"; then
-  machine_hostname=$(ifconfig | grep 'inet ' | grep -v '127' | awk '{ print $2 }' | head -n 1 | cut -d '/' -f 1)
+  machine_hostname=$(ifconfig | grep 'inet ' | awk '{ print $2 }' | grep -v '^127' | head -n 1 | cut -d '/' -f 1)
   if ! check_hostname "${machine_hostname}"; then
     echo "Failed to find an appropriate hostname for the demo."
     exit 1
   fi
+  printf "%b Using Hostname  %s instead \n" ${INFO_EMOJI} "${machine_hostname}"
```

Comment:
Very minor suggestion: I guess that if we are here, then it means that we are using an IP address.

```suggestion
  printf "%b Using IP address %s instead \n" ${INFO_EMOJI} "${machine_hostname}"
```

An alternative suggestion would be to add this line after the `fi` at the line 143 such as:

```bash
printf "%b Using %s as a hostname/IP address\n" ${INFO_EMOJI} "${machine_hostname}"
```

Reply:
Agreed, I've changed it to IP address now :)